index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c3c0d932d2.js" crossorigin="anonymous"></script>
+  <title>Document</title>
+</head>
+<body>
+  <header class="navbar">
+    <i class="fa-solid fa-bars menu-btn" id="menu-btn"></i>
+    <div class="navbar-menu">
+      <div class="menu-options">
+        <ul class="options">
+          <li class="nav-icon"><i class="fa-brands fa-facebook"></i></li>
+          <li class="nav-icon"><i class="fa-brands fa-square-twitter"></i></li>
+          <li class="nav-option">English</li>
+          <li class="nav-option">My page</li>
+          <li class="nav-option">Logout</li>
+        </ul>
+      </div>
+      <nav class="menu-links" id="menu">
+        <i class="fa-regular fa-circle-xmark close-btn menu-btn" id="close-btn"></i>
+        <img src='media/event-logo.png' alt='logo' class='nav-logo' id='nav-logo'>
+        <ul class="nav-links">
+          <li class="nav-link"><a href="#" id="about-link">About</a></li>
+          <li class="nav-link"><a href="#">Program</a></li>
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <div class="home-page active" id="home-page">
+    <section class="home">
+      <h1 class="p-title">YEARLY COMEDIANS WORLD CONFERENCE</h1>
+      <p class="description">A joyful reunion of the most popular living comedians from all over the globe, to celebrate comedy, taking place this December in Spain.</p>
+      <div class="date-and-place">
+        <p class="date">2022.12.15(THU) - 16(FRI)</p>
+        <p class="place">@ Camp Nou, C. d'Arístides Maillol, 12, 08028 Barcelona, Spain</p>
+      </div>
+    </section>
```

Comment:
- [ ] This section has a subtitle at the top followed by the man title in the original design. kindly implement it in your design as well as this is a requirement.

Your design
![image](https://user-images.githubusercontent.com/81932478/203663831-472d589a-fea4-4199-8ae5-1885365d4861.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/203664130-ab31758f-63ba-4002-9d47-051cb38c3dc4.png)